package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "dash-ag-grid",
-    "version": "31.2.1",
+    "version": "31.0.1",
```

Comment:
Should be 31.2.0.

Reply:
I just reverted it, otherwise this will create a draft. 😛

Reply:
A draft what?

Reply:
Of the release, right?

Reply:
The release draft will only be generated when we push a tag

Reply:
Ah, ok. XD